Update active-directory-saas-learnupon-tutorial.md

benwalio opened the pull request:
The full certificate is not necessary for the integration, the Thumbprint is required. Would like to update the documentation to reflect this. I've also created a new screenshot which is attached but will need to be added separately.


<img width="272" alt="tutorial_learnupon_certificate_update" src="https://user-images.githubusercontent.com/13092287/38942672-64f813f6-42fd-11e8-8bf6-46a62ff29632.png">

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @chipalost, has been notified to review your proposed change.

4c74356b41 commented:
Just a ping ;)

dominicbetts commented:
@4c74356b41 - it looks like there's now a merge conflict on this PR - can you fix the merge conflict and then we can accept your change?
Thanks, Dominic

dominicbetts commented:
#please-close
@4c74356b41 @chipalost - rather than you fixing the merge conflict - I've made the change directly.

Commit:
Update active-directory-saas-learnupon-tutorial.md

The full certificate is not necessary for the integration, the Thumbprint is required. Would like to update the documentation to reflect this.
